openedx/adg/lms/applications/models.py

```diff
@@ -48,9 +48,11 @@ def set_is_prerequisite_courses_passed(self):
 
     def set_is_written_application_completed(self):
```

Comment:
Maybe update the method name to reflect the changes in method.

`set_is_written_application_completed_with_date` is an option.